From linux-usb-devel-bounces@lists.sourceforge.net Sun Dec 17 13:05:24 2006
From: Johannes Hölzl <johannes.hoelzl@gmx.de>
To: linux-usb-devel@lists.sourceforge.net
Date: Sun, 17 Dec 2006 22:05:09 +0100
Message-Id: <1166389509.5105.46.camel@localhost.localdomain>
Mime-Version: 1.0
Subject: USB: Bugfix for aircable: Add module and name to usb_serial_driver

From: Johannes Hölzl <johannes.hoelzl@gmx.de>

While adding the dynamic-id support to usb serial I found a small bug in
the air cable driver:

Adds module and name information to the usb_serial_driver instance
of aircable. So the aircable driver is correctly shown under
/sys/bus/usb-serial/drivers/aircable and has the module link.

Signed-off-by: Johannes Hölzl <johannes.hoelzl@gmx.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/aircable.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/serial/aircable.c
+++ gregkh-2.6/drivers/usb/serial/aircable.c
@@ -581,7 +581,10 @@ static struct usb_driver aircable_driver
 };
 
 static struct usb_serial_driver aircable_device = {
-	.description =		"aircable",
+	.driver = {
+		.owner =	THIS_MODULE,
+		.name =		"aircable",
+	},
 	.usb_driver = 		&aircable_driver,
 	.id_table = 		id_table,
 	.num_ports =		1,
